OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -477,6 +492,7 @@ public DiscussionThreadViewHolder(View itemView) {
                     findViewById(R.id.discussion_responses_thread_row_body_text_view);
             threadVisibilityTextView = (TextView) itemView.
                     findViewById(R.id.discussion_responses_thread_row_visibility_text_view);
+            actionsBar = itemView.findViewById(R.id.discussion_actions_bar);
```

Comment:
This should probably be at the start of the views, since it's the root. Similarly in the other view holders.